src/opm/input/eclipse/Schedule/Schedule.cpp

```diff
@@ -529,25 +529,37 @@ namespace
 /// \brief Check whether each MS well has COMPSEGS entry andissue error if not.
 /// \param welsegs All wells with a WELSEGS entry together with the location.
 /// \param compegs All wells with a COMPSEGS entry
-void check_compsegs_consistency(::Opm::Schedule::WelSegsSet& welsegs, std::set<std::string>&  compsegs)
+void check_compsegs_consistency(::Opm::Schedule::WelSegsSet& welsegs, 
+                                std::set<std::string>&  compsegs,
+                                const std::vector<::Opm::Well>& wells)
 {
     std::vector<std::pair<std::string,::Opm::KeywordLocation>> difference;
     difference.reserve(welsegs.size());
     std::set_difference(welsegs.begin(), welsegs.end(),
                         compsegs.begin(), compsegs.end(),
                         std::back_inserter(difference),
                         ::Opm::Schedule::PairComp());
+    // Ignore wells without connections
+    const auto empty_conn = [wells](const std::pair<std::string,::Opm::KeywordLocation> &x) -> bool {
```

Comment:
Tiny nit: This object capture (`[wells]`) will make a copy of the `wells` vector inside the lambda.  If you write `[&wells]` with an ampersand instead you'll capture the `wells` vector by reference and that's okay since the lambda does not live beyond the scope of `check_compsegs_consistency()` and does not mutate the `wells` either.

Reply:
Ah, thanks - neglect from my side, will update.